### ISSUE TITLE
yml/spaced-comment false positive and fixing increases spaces

### ISSUE THREAD

**Issue by leotm** [ota-meshi/eslint-plugin-yml#91]
ESLint https://github.com/leotm/react-native-template-typescript/blob/master/.eslintrc.js ([disabled the rule for now](https://github.com/leotm/react-native-template-typescript/commit/8b877e1210bee02c138ce060a40e192422f14025))

YAML https://github.com/leotm/react-native-template-typescript/blob/master/.github/workflows/main.yml

Current

![Screenshot 2021-07-19 at 19 15 56](https://user-images.githubusercontent.com/1881059/126207520-49f70ab5-7577-4a52-9e29-c6955956865b.png)

Save

![Screenshot 2021-07-19 at 19 16 06](https://user-images.githubusercontent.com/1881059/126207542-27b92106-99c8-475d-95a8-b2137d2fb94e.png)

Save again

![Screenshot 2021-07-19 at 19 16 48](https://user-images.githubusercontent.com/1881059/126207629-75bc59d9-96f5-4283-b8a6-0f9645a45147.png)

Great plugin so far, keep up the good work

**Comment by ota-meshi** [comment]
Thank you for posting this issue.

You need to use `yml/spaced-comment` instead of `spaced-comment`.

```yaml
overrides: 
  - 
    files: ["*.yaml", "*.yml"]
    rules: 
        spaced-comment: off
        yml/spaced-comment: error
```

I have updated the documentation that explains it.

https://ota-meshi.github.io/eslint-plugin-yml/rules/spaced-comment.html

**Comment by leotm** [comment]
Np, cheers for updating the docs, JS syntax would be preferable here for `.eslintrc.js`

**Comment by leotm** [comment]
Here's a PR https://github.com/ota-meshi/eslint-plugin-yml/pull/92, feel free to close if sticking to _yaml_ syntax
